### PR TITLE
Update README for EC2 boto3 requirement

### PR DESCRIPTION
**Pull request by taqtiqa-admin**
TASK [ec2-security-group : Create the EC2 security group] **********************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "boto3 required for this module"}
	to retry, use: --limit @/home/<snip>/src/streisand/playbooks/amazon.retry

PLAY RECAP *********************************************************************
localhost                  : ok=2    changed=0    unreachable=0    failed=1

### CODE REVIEW

**Comment by nopdotcom** [comment]
Could you file a bug report describing the problem? Thanks!

**Comment by gallaghersart** [comment]
I ran `pip install boto3` to instll boto3
that fixed that problem
but now I'm left with this problem, I was able to login into a server I tried last night, then started from scratch today and getting same error
thanks
`
TASK [set_fact] ********************************************************************************************************
ok: [x.x.x.32]

PLAY [Checking instance status] ****************************************************************************************

TASK [Wait for cloud-init to complete] *********************************************************************************
fatal: [x.x.x.32]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: Warning: Permanently added 'x.x.x.32' (ECDSA) to the list of known hosts.\r\nLoad key \"/home/ubuntu/.ssh/id_rsa\": Permission denied\r\nPermission denied (publickey).\r\n", "unreachable": true}
        to retry, use: --limit @/home/ubuntu/streisand/playbooks/amazon.retry

PLAY RECAP *************************************************************************************************************
x.x.x.32               : ok=1    changed=0    unreachable=1    failed=0
localhost                  : ok=24   changed=15   unreachable=0    failed=0
`

**Comment by gallaghersart** [comment]
I re-ran certs, updates and rebooted all ok now html file did not create so i ran `sudo apt-get install --reinstall xdg-utils`

**Comment by cpu** [comment]
> I re-ran certs, updates and rebooted all ok now html file did not create so i ran sudo apt-get install --reinstall xdg-utils

@gallaghersart The message about xdg-utils is for a convenience step that tries to open the generated HTML in your web browser. It's optional and failure to open the docs this would should only produce a warning.

**Comment by nopdotcom** [comment]
For what it's worth, I started seeing the `TASK [Wait for cloud-init to complete]` bomb out several times on both DO and EC2 around the time you reported this. It all seems to work today, which is sorta disappointing, but probably good.

**Comment by nopdotcom** [comment]
For what it's worth, I started seeing the `TASK [Wait for cloud-init to complete]` bomb out several times on both DO and EC2 around the time you reported this. It all seems to work today, which is sorta disappointing, but probably good.

File: README.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ Complete all of these tasks on your local home machine.
     you can skip this section.
   * Amazon EC2
 
-        sudo pip install boto
```
Comment (cpu) [inline README.md]: In my testing **both** `boto` and `boto3` are required. Can you change this to: `sudo pip install boto boto3`?

**Comment by cpu** [comment]
@taqtiqa-admin Bumping this PR again. I left some minor feedback - will you have a chance to address it? If not since this is a very small PR I'd prefer to fix it myself versus wait too much longer.

Thanks!

**Comment by cpu** [comment]
Closing in favour of https://github.com/StreisandEffect/streisand/pull/1163 - Thanks for flagging the issue!

**Pull request closed without merging**